src/ansys/fluent/core/post_objects/timing_decorator.py

```diff
@@ -0,0 +1,16 @@
+"""Provides a module for timing function decorator."""
+
+import time
+
+
+def timing(func):
+    """Timing function decorator."""
```

Comment:
Description should start with a verb ending in "s" or "es". Describe what the function does.

Reply:
Done. Thank you.